nautobot/docs/development/core/getting-started.md

```diff
@@ -632,20 +632,40 @@ Unit tests are run using the `invoke unittest` command (if using the Docker deve
+##### Useful Unit Test Parameters
 
-| Docker Compose Workflow    | Virtual Environment Workflow                                                             |
-| -------------------------- | ---------------------------------------------------------------------------------------- |
-| `invoke unittest --keepdb` | `nautobot-server --config=nautobot/core/tests/nautobot_config.py test --keepdb nautobot` |
+The `invoke unittest` command supports a number of optional parameters to influence its behavior. Careful use of these parameters can greatly reduce the time it takes to run and re-run tests during development.
 
-!!! note
-    Using the `--keepdb` argument will raise errors if you've modified any model fields since the previous test run.
+* `--cache-test-fixtures` - Cache [test factory data](./testing.md#factory-caching) to disk, or if a cache is already present, load from that cache when initializing the test environment. Can significantly improve the initial startup time of the test suite after your first test run.
+* `--failfast` - Fail as soon as any test failure or error condition is encountered, instead of running to completion.
+* `--keepdb` - Save and reuse the initialized test database between test runs. Can significantly improve the initial startup time of the test suite after your first test run. **Do not use if you're actively making changes to model definitions or migrations.**
+* `--label <module.path>` - Only run the specific subset of tests. Can be broad (`--label nautobot.core.tests`) or specific (`--label nautobot.core.tests.test_graphql.GraphQLQueryTestCase`).
+* `--no-buffer` - Allow stdout/stderr output from the test to be seen in your terminal, instead of being hidden. **If you're debugging code with `breakpoint()`, you should use this option, as otherwise you'll never see the breakpoint happen.**
+* `--parallel` - Split the tests across multiple parallel subprocesses. Can greatly reduce the runtime of the entire test suite when used.
+* `--skip-docs-build` - Skip building/rebuilding the static Nautobot documentation before running the test. Saves some time on reruns when you haven't changed the documentation source files.
+* `--verbose` - Run tests more verbosely, including describing each test case as it is run.
 
-!!! warning
-    In some cases when tests fail and exit uncleanly it may leave the test database in an inconsistent state. If you encounter errors about missing objects, remove `--keepdb` and run the tests again.
+##### Unit Test Invocation Examples
 
-+/- 1.5.11
-    The `--cache-test-fixtures` argument was added to the `invoke unittest` and `nautobot-server test` commands to allow for caching of test factory data between test runs. See the [factories documentation](./testing.md#factory-caching) for more information.
+In general, when you first run the Nautobot tests in your local copy of the repository, we'd recommend:
+
+```no-highlight
+invoke unittest --cache-test-fixtures --keepdb --parallel
```

Comment:
Long term I'd love to see us to make `cache-test-fixtures` and `parallel` be on by default as opt-out parameters instead.